components/logins/src/login.rs

```diff
@@ -125,7 +425,10 @@ impl Login {
 
             time_password_changed: row.get("timePasswordChanged")?,
             times_used: row.get("timesUsed")?,
-        })
+        };
+        // For now, we want to apply fixups but still return the record if
+        // there is unfixably invalid data in the db.
+        Ok(login.maybe_fixup().unwrap_or(None).unwrap_or(login))
```

Comment:
I think we'd eventually like to be confident our db only holds validated data, and not have to do this on every read. But that seems like a much later step in the process.